src/components/Modal/README.md

```diff
@@ -92,11 +92,19 @@ See [API](#api) for all available options.
 - **Modal actions** should correspond to the modal purpose, too. E.g. “Delete”
   tells better what happens rather than “OK”.
 
-- Modal **automatically focuses the first non-disabled form field** by default
-  which allows users to confirm the modal by hitting the enter key. When no
-  field is found then the primary button (in the footer) is focused. To turn
+- Modal **automatically focuses the first non-disabled form field** by default.
+  When no field is found then the primary button (in the footer) is focused. To turn
   this feature off, set the `autofocus` prop to `false`.
 
+- Modal **submits the form when the user presses the `Enter` key** . The primary
+  button is clicked in this case. To turn this feature off, set the
```

Comment:
I think the click is just _emulated_, isn't it?

Reply:
It is not. `click()` function is called directly on HTML button, it is not emulated, it's real click that also triggers its own click event. So I would not call it emulated.

@mbohal can correct me, but emulated event would be dispatchEvent(event), but not click().

Reply:
> A click is programmatically triggered on the primary  button in this case.

This is great. Thanks!